tests/test_pylint_runners.py

```diff
@@ -57,6 +59,28 @@ def test_runner_with_arguments(runner: _RunCallable, tmpdir: LocalPath) -> None:
         assert err.value.code == 0
 
 
+def test_pylint_argument_deduplication(tmpdir: LocalPath) -> None:
+    """Check that the Pylint runner does not over-report on duplicate
+    arguments.
+
+    (GitHub #6242, #4053).
+    """
+    filepath = str(
+        pathlib.Path(__file__)
+        .resolve()
+        .parent.joinpath("functional", "t", "too", "too_many_branches.py")
+    )
```

Comment:
```suggestion
def test_pylint_argument_deduplication(tmpdir: LocalPath, tests_directory: Path) -> None:
    """Check that the Pylint runner does not over-report on duplicate
    arguments.

    See https://github.com/PyCQA/pylint/issues/6242 and
    https://github.com/PyCQA/pylint/issues/4053
    """
    filepath = str(tests_directory / "functional/t/too/too_many_branches.py")
```

Reply:
Ah, I didn't think `"functional/t/too/too_many_branches.py"` was compatible with Windows, which is why I used the `joinpath` method. But, I guess `pathlib` is smart enough to do the right thing. TIL. Thanks.

Reply:
Yeah it works on Windows. The "issue" with my suggestion was that Path was not imported 😄